### PR TITLE
Rename llvm_stable feature and remove references from docs.

### PR DESCRIPTION
**Pull request by faineance**
Fixes: https://github.com/servo/rust-bindgen/issues/409

### CODE REVIEW

**Comment by fitzgen** [comment]
Awesome! Taking a look now :)

File: bindgen-integration/Cargo.toml [hunk 1]
```diff
@@ -11,4 +11,4 @@ bindgen = { path = ".." }
 gcc = "0.3"
 
 [features]
-llvm_stable = ["bindgen/llvm_stable"]
+__testing_only_llvm_stable = ["bindgen/__testing_only_llvm_stable"]
```
Comment (fitzgen) [inline bindgen-integration/Cargo.toml]: IIRC, `cargo` had issues with `_` prefixed features, so we may need to change this to simply `testing_only_llvm_stable`. We'll see when travis CI results get in :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Run `cargo test` to compare generated Rust bindings to the expectations.
 ### Running All Tests
 
 ```
-$ cargo test [--features llvm_stable]
+$ cargo test --all-features
```
Comment (fitzgen) [inline CONTRIBUTING.md]: Oh neat! TIL

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ that you aren't forgetting to document types and functions. CI will catch it if
 you forget, but the turn around will be a lot slower ;)
 
 ```
-$ cargo build --features "llvm_stable _docs"
+$ cargo build --features docs_
```
Comment (fitzgen) [inline CONTRIBUTING.md]: Yeah the trailing `_` was because leading ones weren't properly handled. Perhaps this should be `testing_only_docs` too?

**Comment by fitzgen** [comment]
Ping me after removing leading underscores and I'll r+

Thanks again!

**Comment by faineance** [comment]
Removed the leading underscores, now the build will tell.

**Comment by fitzgen** [comment]
Thanks!

@bors-servo r+

**Pull request merged**